iphone/Classes/XMLModule.m

```diff
@@ -12,6 +12,15 @@
 
 @implementation XMLModule
 
+- (id)_initWithPageContext:(id<TiEvaluator>)context
+{
+  if (self = [super _initWithPageContext:context]) {
+    DebugLog(@"[WARN] The Ti.XML namespace has been deprecated in favor of a standalone module in SDK 8.0.0.");
```

Comment:
We should try and use a more formal way of marking the code as deprecated too: https://stackoverflow.com/questions/1443599/how-do-i-flag-a-function-as-being-deprecated-in-an-iphone-objective-c-header-fil

So, I guess in this case that'd mean marking all of the XML module/types methods/properties in the header(s) as deprecated?